spring-cloud-stream-app-generator/pom.xml

```diff
@@ -154,6 +154,25 @@
 						</syslog-source>
 						<task-launcher-local-sink />
 						<task-launcher-yarn-sink />
+						<task-launcher-cloudfoundry-sink>
+							<forceDependencies>
+								<dependency>
+									<groupId>io.projectreactor</groupId>
+									<artifactId>reactor-core</artifactId>
+									<version>3.0.2.RELEASE</version>
+								</dependency>
+								<dependency>
+									<groupId>io.projectreactor.ipc</groupId>
+									<artifactId>reactor-netty</artifactId>
+									<version>0.5.1.RELEASE</version>
+								</dependency>
+								<dependency>
+									<groupId>io.netty</groupId>
```

Comment:
Interesting. SCDF server only re-overrides reactor and reactor-netty:

https://github.com/spring-cloud/spring-cloud-deployer-cloudfoundry/blob/master/pom.xml#L23-L24